static_src/components/user_list.jsx

```diff
@@ -125,16 +129,22 @@ export default class UserList extends React.Component {
   }
 
   render() {
+    let saving;
     let loading = <Loading text="Loading users" />;
     let content = <div>{ loading }</div>;
 
+    if (this.props.saving) {
```

Comment:
Since we also do an if check in the `Loading` component to decide what to render, what do you think about removing this `if` statement and replacing the contents of `saving` with
```
<Loading loadingDelayMS={`${0}`} text="Saving" style="globalSaving" active={this.props.saving} />
```